Tools/environment_install/installROS-ubuntu.sh

```diff
@@ -0,0 +1,230 @@
+#!/bin/bash
+echo "---------- $0 start ----------"
+set -e
+# set -x
+
+ROS_WS_ROOT=$HOME/ardupilot-ws
+
+red=`tput setaf 1`
+green=`tput setaf 2`
+reset=`tput sgr0`
+
+function heading() {
+    echo "$sep"
```

Comment:
sep ?

Reply:
These sort of reviews is why PRs never get merged, this is directly from another script https://github.com/ArduPilot/ardupilot/blob/4b20a2d5f1fee80aa2ee611e75ef63c7d5eef125/Tools/environment_install/install-prereqs-ubuntu.sh#L47